Assets/Scripts/PickupStaminaBoost.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PickupStaminaBoost : MonoBehaviour {
```

Comment:
I disagree, this is the behaviour for the particular pickup. the object itself isnt responsible for been picked up, the player is, the object simply makes its alterations. despite what you may think I have a design in mind